src/js/main.js

```diff
@@ -618,7 +629,7 @@ $(function () {
      * @return {boolean}
      */
     function CanBuildHQ(x, y){
-        return !CheckAdjacentBuilding(x, y, [builds.HQ, builds.GOLD, builds.COAL, builds.OIL, builds.IRON, builds.BAUXITE, builds.LEAD, builds.SULFUR, builds.NITER, builds.STONE]);
+        return !CheckAdjacentBuilding(x, y, [builds.HQ, builds.GOLD, builds.COAL, builds.OIL, builds.IRON, builds.BAUXITE, builds.LEAD, builds.SULFUR, builds.NITER, builds.STONE, builds.ROCK]);
```

Comment:
Pokud znemožníme hráčům postavit HQ hned vedle skal (což asi ani není nutné), tak ať je to i server-side. Buď obojí nebo ani jedno.